### PR TITLE
Awesome Books With Modules

### PR DESCRIPTION
**Pull request by Chimoneg27**
### For this activity I did the following

- Made sure to remove all linter errors
- Added the luxon module
- Made sure current date and time is displayed below the nav bar
- Converted my functions to arrow functions
- Created a module folder with js files for exporting 
- Linked all the module js files in the index.js file

### CODE REVIEW

File: modules/navbar.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
+export default function Navbar() {
+  const bookForm = document.getElementById('book-entry');
```
Comment (AbelG101) [inline modules/navbar.js]: - [ ] You're doing good so far! However, as part of the project requirements you should refactor your function expressions to arrow functions so as such please replace the Navbar() function expression with the arrow function shown below 👇.
```js
const Navbar = () => {
  // ...your code goes here
}
```

**Review by AbelG101 — CHANGES_REQUESTED** [review]
# Hi @Chimoneg27 👋, 

You're doing a good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Proper project setup ✔
- Proper code modularisation ✔
- Current date is properly displayed ✔
- Correct use of GitHub Flow ✔
- All linter checks are passing ✔
- Descriptive PR title with a great summary ✔

## Required Changes ♻️
- _Check the comments under the review._
- you just have to make a few changes and you're on your way 🚀.


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: modules/navbar.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    bookForm.style.display = 'none';
+    contacts.style.display = 'block';
+  };
+
+  contactClick.addEventListener('click', contactUs);
+  addClick.addEventListener('click', showForm);
+  listClick.addEventListener('click', showShelf);
+};
+
+export default Navbar;
```
Comment (Gambit142) [inline modules/navbar.js:44]: - [ ] Great work changing your functions to use the arrow syntax. However, I noticed that when I click on a link, the relevant part of the page does not show. Kindly fix this. 👍 

chrome-capture-2023-7-14

**Review by Gambit142 — CHANGES_REQUESTED** [review]
Hi @Chimoneg27 👋, 

Great work on making the changes requested by a previous reviewer 👏 

## Highlights
- Set up linters correctly ✔️ 
- Good use of arrow syntax to create functions ✔️ 
- Good use of JavaScript modules ✔️ 
- Good use of let and const keywords ✔️ 

You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were simply missed by the previous reviewer and addressing them will improve your application._


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### Please tag me in your question so I can receive the notification. You can tag me by typing @Gambit142

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Nemwel-Boniface — APPROVED** [review]
## Hello @Chimoneg27   👋,

I am Nemwel your reviewer.


## _This is an additional review as a result of the second opinion request._

I have received your second opinion request with a complaint about the previous reviewer @Gambit142 . Below are my findings:

For some reason when I test your project locally, everything seems to be working perfectly fine as it is supposed to and it is strange why the project does not work on @Gambit142 's end. The only explanation is that he might have been testing the project in an online server instead of testing it locally.

Below is how I am seeing your project when I test it locally:

[Screencast from 14-08-2023 06:02:20 ALASIRI.webm](https://github.com/Chimoneg27/Awesome-Books-ES6/assets/86318284/dbf1fadd-4d16-4bf5-b2e9-370de26dba4a)

From my end, your project was ready for approval. :rocket: 

As this was the only reason that your project was not approved it will be approved as soon as possible.

Please note that since we will approve it manually so this could take a few hours!

In case anything is not 100% clear tag me @Nemwel-Boniface so that I get the notification.

Keep rocking 🚀!

**Comment by Gambit142** [comment]
Hi @Nemwel-Boniface, I tested it locally and it didn't work, and not on an online server. That was the reason why I attached the gif. It is great that it works on your side but it didn't on mine, that was the reason why I requested the change. Thank you for the extra review, I just wanted to clarify this. 👍

**Pull request merged**